UPGRADE.md

```diff
@@ -284,6 +297,16 @@ sulu_admin:
             - "%kernel.project_dir%/config/forms"
 ```
 
+The datagrids will be handled equally and the configuration inside `sulu_admin.resources.categories.datagird` will be
```

Comment:
I don't think what you've written here is true... The configuration you are mentioning here is not being replaced by the one below. What the configuration below allows is to add your own datagrids in your project, which wasn't possible before.